tests/unit-tests/unsupported-theme-handlers/test-class-sensei-unsupported-theme-handler-course-archive.php

```diff
@@ -116,6 +116,27 @@ public function testShouldSetupDummyPost() {
 		$this->assertEquals( 'page', $post->post_type, 'The dummy post type should be "page"' );
 	}
 
+	public function testCourseArchivePage_WhenRendered_DoesNotRenderQueryListBlockWithoutPageContent() {
+		/* ACT */
+		$this->handler->handle_request();
+
+		/* ASSERT */
+		global $post;
+		$this->assertStringNotContainsString( 'wp-block-sensei-lms-course-list', $post->post_content );
+	}
+
+	public function testCourseArchivePage_WhenRendered_DoesNotRenderQueryListBlockIfPageContentAvailable() {
```

Comment:
It kinda looks like the assertion in this test case is showing the block on the page?

Reply:
Good catch! Updated the name here https://github.com/Automattic/sensei/pull/6354/commits/a1cd4d7639d5830f147ad5231d00e16cb1bce200